API/Models/AccountModels/RegisterModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
Maybe we should call this class `CreateUserDTO` according to what data and where from it comes?

Reply:
I've combined RegisterModel and LoginModel into AuthDTO

Reply:
Maybe we could also move this class out of `AccountModels` folder as well just to be consistent?

Reply:
Yes, moved.